content/en/docs/user-documentation/how-tos/metal-lb.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
This is also adding metal-lb :-)

@fzdarsky was wondering if we must yet talk about metal-lb, we need to decide how we will handle the master VIPs, if this or a more compact/featureful library.

as a how-to that looks good to me anyway. I'm not against it (and also I tested it and works).

Reply:
I agree a PR should focus on a single thing, here `s/kubectl/oc/g`, so if this is also adding a new HowTo (instead of just updating) it should be in a separate PR.

Re MetalLB: It's true that we may opt for a different load balancing mechanism and that we said HowTos should only include recommended ways of using MicroShift. Then again, this addresses a concrete need and we can still update our recommendation later.

Reply:
@fzdarsky im going to remove metal-lb and will start convo in different PR on how to proceed